roou.py

```diff
@@ -10,25 +10,28 @@
 from config import Config as conf
 import normalizing
 import astropy.table as astrotable
+import logging
 
 parser = argparse.ArgumentParser()
-# Path to directory where SExtractor should save the temporary files.
-tmpdir_for_SExtractor = '/mnt/ds3lab/dostark/z_'+str(conf.redshift)+\
-                        '/r-band/tmp_for_SExtractor_'+str(conf.ext)+\
-                        '_'+str(conf.stretch_type)+'_'+\
-                        str(conf.scale_factor)+'/'
+
+# Path to the 'read_PSF' SDSS PSF tool : http://www.sdss.org/dr12/algorithms/read_psf/
+psfTool_path = ''
+if psfTool_path == '':
+    logging.warn('In roou.py : no path provided to your SDSS PSF tool.'\
+                 'This might raise an error if trying to add a SDSS PSF')
+
 # Path to PSF metadata (psFields) which the SDSS PSF tool uses as input.
-psfTool_path = '/mnt/ds3lab/blaunet/readAtlasImages-v5_4_11/read_PSF'
-psfFields_dir = '/mnt/ds3lab/galaxian/source/sdss/dr12/psf-data'
-# Path to Hubble PSF if used:
+psfFields_dir = conf.core_path+'/source/sdss/dr12/psf-data'
 
-# The following paths are only needed, if the Hubble PSF is used for simulating 
+# Path to Hubble PSF if used:
+# The following paths are only needed, if the Hubble PSF is used for simulating
 # AGN point sources.
 path_to_Hubble_psf = '/mnt/ds3lab/dostark/hubble_psf_many_stars.fits'
 #path_to_Hubble_psf = '/mnt/ds3lab/dostark/PSFSTD_WFC3IR_F160W.fits'
-tmp_GALFIT = '/mnt/ds3lab/dostark/hubble_z_'+str(conf.redshift)+\
-             '/'+conf.filter_+'-band/tmp_for_GALFIT_'+str(conf.ext)+\
-             '_'+str(conf.stretch_type)+'_'+str(conf.scale_factor)+'/'
+
+# Directories for temporary files
+tmpdir_for_SExtractor = conf.stretch_setup+'/tmp_for_SExtractor/'
+tmp_GALFIT = conf.stretch_setup+'/tmp_for_GALFIT/'
```

Comment:
@dominicstark Is it a problem to change the paths of these temporary directories like that ? Information is the same, just organized in a different way